packages/babel-parser/test/expressions/is-expression-babel-parser/fail/8/output.json

```diff
@@ -1,20 +1,14 @@
 {
   "type": "Identifier",
-  "start": 1,
-  "end": 7,
-  "loc": {
-    "start": {
-      "line": 2,
-      "column": 0
-    },
-    "end": {
-      "line": 2,
-      "column": 6
-    },
-    "identifierName": "public"
-  },
+  "start":1,"end":7,"loc":{"start":{"line":2,"column":0},"end":{"line":2,"column":6},"identifierName":"public"},
   "name": "public",
   "errors": [
-    "SyntaxError: Unexpected reserved word 'public' (2:0)"
+    {
+      "$$ babel internal serialized type": "Error",
+      "message": "Unexpected reserved word 'public' (2:0)",
```

Comment:
I like the original error serialized by `error.name: error.message`.

I think a single unit test asserting the shape of actual error is good enough. The stability of `code` and `reasonCode` should be respected in review process instead of in fixtures.

Reply:
Agreed!

Reply:
I dropped commits related to updating snapshots.